### DISCUSSION TITLE
wxid加好友

### DISCUSSION THREAD

**Discussion by pwh-pwh** [ttttupup/wxhelper#92] [Q&A]
想问下大佬wx_id加好友是如何实现的

**Comment by ttttupup** [comment]
不是实现的，是微信就有的功能

**Reply by pwh-pwh** [reply]
pc端我不清楚，大佬研究mac端或者安卓端吗，很想实现一下

**Reply by ttttupup** [reply]
没mac呀，安卓的就更简单了，直接用frida很方便

**Reply by pwh-pwh** [reply]
> 没mac呀，安卓的就更简单了，直接用frida很方便

大佬，安卓有空可以搞吗，或者提供思路也可以

**Reply by ttttupup** [reply]
> > 没mac呀，安卓的就更简单了，直接用frida很方便
> 
> 大佬，安卓有空可以搞吗，或者提供思路也可以

安卓的没时间。弄到最后就是过风控。

**Reply by pwh-pwh** [reply]
换个思路wxid转wx号有没有可能实现

**Comment by dadi-cloud** [comment]
之前有个网站可以使用wx_id转成二维码添加， 不过那个网站不能用了

**Comment by ttttupup** [comment]
移动端风控才是难点

**Comment by pwh-pwh** [comment]
3.9.5.81版本的可以添加下wxid加好友功能吗，有些功能后面版本的没有了
